assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -6148,9 +6143,9 @@ tests:
                 - organization#member
               object: repo:openfga/openfga
               relation: reader
-            temporarilySkipReason: "because userset relations not being returned properly, `organization:openfga` returned"
+            temporarilySkipReason: "because userset relations not being returned properly" # fails check, review if assertion is correct
             expectation:
-              - organization:openfga#member
+              - organization:openfga#member # is this correct?
```

Comment:
It's not. I came to the same conclusion in https://github.com/openfga/openfga/pull/1491/files

:)